packages/flat-components/src/containers/CloudStorageContainer/CloudStorageContainer.stories.tsx

```diff
@@ -265,7 +274,13 @@ export const Overview: Story<FakeStoreConfig> = config => {
     const [store] = useState(() => new FakeStore(config));
     return (
         <div className="ba br3 b--gray overflow-hidden" style={{ height: 600, maxHeight: "80vh" }}>
-            <CloudStorageContainer store={store} />
+            <CloudStorageContainer
+                path={"/new directory/directory1/directory2/"}
```

Comment:
```suggestion
                path={"/path/to/directory/"}
```

Let's avoid whitespace in the path for a better visual.